### PR TITLE
Update rem.md

### PR DESCRIPTION
**Pull request by npocmaka**
multi-line comments.

### CODE REVIEW

**Comment by PRMerger7** [comment]
@npocmaka : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: WindowsServerDocs/administration/windows-commands/rem.md [hunk 1]
```diff
@@ -13,6 +13,9 @@ ms.date: 10/16/2017
 
 Records comments in a script, batch, or config.sys file. If no comment is specified, **rem** adds vertical spacing.
 
+> [!NOTE]
+> This command is internal for cmd.exe .
```
Comment (illfated) [inline WindowsServerDocs/administration/windows-commands/rem.md:17]:
```suggestion
> This command is internal to the command-line interpreter, cmd.exe .
```

File: WindowsServerDocs/administration/windows-commands/rem.md [hunk 2]
```diff
@@ -72,6 +75,17 @@ create partition logical
...
 ```
 
+For multi-line comments use conditional execution:
+
+```
+  Rem/||(
+    The REM evaluates to success so these
+    lines will never be executed.
+    Mind that here you need to escape closing parentheses
+    within multi-line comment blocks like this ^)
```
Comment (illfated) [inline WindowsServerDocs/administration/windows-commands/rem.md:85]:
```suggestion
    The REM statement evaluates to success,
    so these lines will never be executed.
    Keep in mind that you will need to escape closing parentheses
    within multi-line comment blocks like this: `^)`
```

**Comment by illfated** [comment]
My suggestions are made in the hope that you will find them useful. Feel free to discuss the details with me, in case you disagree or find them difficult to understand.

**Comment by PRMerger15** [comment]
@npocmaka : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by npocmaka** [comment]
> My suggestions are made in the hope that you will find them useful. Feel free to discuss the details with me, in case you disagree or find them difficult to understand.

thanks for the proposals. As not native speaker myself they were useful for me.

**Review by illfated — APPROVED** [review]
Looks good to me. Thank you for contributing.

**Comment by illfated** [comment]
#sign-off

**Comment by PRMerger9** [comment]
@illfated: I'm sorry - only the author of this article, @eross-msft, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

**Comment by illfated** [comment]
Yes, I know. I made the sign-off comment based on an existing agreement with the team.

**Comment by PRMerger19** [comment]
@sethmanheim : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger13** [comment]
@sethmanheim : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Pull request merged**